experimental/examples/imageviewer/shared/src/androidMain/kotlin/example/imageviewer/AndroidImageStorage.kt

```diff
@@ -0,0 +1,63 @@
+package example.imageviewer
+
+import androidx.compose.runtime.snapshots.SnapshotStateList
+import androidx.compose.ui.graphics.ImageBitmap
+import androidx.compose.ui.graphics.asAndroidBitmap
+import androidx.compose.ui.graphics.asImageBitmap
+import androidx.core.graphics.scale
+import example.imageviewer.model.PictureData
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.launch
+
+private const val maxStorableImageSize = 2000
+private const val storableThumbnailSize = 200
```

Comment:
It's not super clear what side is meant, let's clearly mention "px"
```suggestion
private const val maxStorableImageSizePx = 2000
private const val storableThumbnailSizePx = 200
```

Reply:
Done